Makefile

```diff
@@ -39,12 +50,26 @@ upgrade:
 	$(CARGO) upgrade
 	$(CARGO) update
 
+unit-test:
```

Comment:
You might be able to use `.ONESHELL` here.
[.ONESHELL](https://www.gnu.org/software/make/manual/html_node/Special-Targets.html#index-recipe-execution_002c-single-invocation)

Reply:
oh let me try!!!